docs/examples/patterns/notifications/notifications.html

```diff
@@ -8,12 +8,17 @@
   <p class="p-notification__response">
     We use cookies to improve your experience. By your continued use of this site you accept such use.
   </p>
-  <button class="p-icon--close" aria-label="Close notification" onclick="closeNotification('notification')">Close</button>
+  <button class="p-icon--close" aria-label="Close notification" data-close="notification">Close</button>
 </div>
 
 <script>
-function closeNotification(target) {
-  var notification = document.getElementById(target);
-  notification.style.display = 'none';
-}
+  function closeNotification(id) {
+    var notification = document.getElementById(id);
+    notification.style.display = 'none';
+  }
+
+  notification.querySelector('[data-close]').addEventListener('click', function(event) {
+    var target = event.target.getAttribute('data-close');
+    closeNotification(target);
```

Comment:
Do you need a function for this?

Reply:
There was a function before, but sure, now when we don't call it from `onclick` it could be inline handler.